lib/models/movie.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const { id } = require('prelude-ls');
+const Bookshelf = require('../libraries/bookshelf');
+
+module.exports = Bookshelf.Model.extend({
+  tableName: 'movies',
+  serialize: function () {
+    return {
+      id: this.get(id),
```

Comment:
id should be inside quotes correct?

Reply:
Yes, I thought so, but for some reason line 3 was generated when I installed hapi-bookshelf-serializer. So I just used that id. But I'll keep it to what the doc says and remove line 3 and keep id inside quotes.